app/views/components/gds/exportsInputTextArea.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.govukfrontend.views.html.components._
+@import views.components.gds.Styles._
+
+@this(govukTextarea : GovukTextarea)
+
+@(
+        field: Field,
+        labelKey: String,
+        hintKey: Option[String] = None,
+        isPageHeading: Boolean = false,
+        headingClasses: String = gdsPageLabel,
+        labelClasses: String = "govuk-label--m"
+)(implicit messages: Messages)
+
+@buildLabel = @{
+    if(isPageHeading) {
+        Label(content = Text(messages(labelKey)), isPageHeading = true, classes = headingClasses)
```

Comment:
can we pass here just pageHeading, and remove if ?

Reply:
The classes are different as well.   Copied this from almost identical `exportsInputText` and didn't look too closely at the logic I'm afraid.